### ISSUE TITLE
Using helm-extra-args: --timeout 600s in config file causes "unknown flag"

### ISSUE THREAD

**Issue by HerrDerb** [helm/chart-testing-action#135]
With version 2.6.0 when using a config file for linting (`ct lint --config ct.yaml`) that includes 
`helm-extra-args: --timeout 600s`

linting fails in the end with
 ```
Validation success! 👍
Error: failed linting charts: failed processing charts
Error: unknown flag: --timeout
```

This was not an issue before 2.5.0

Used Helm version: `v3.10.3`
Plain integration of `chart-testing-action@2.6.0` without any `with`

**Comment by cpanato** [comment]
fix here: https://github.com/helm/chart-testing/pull/605

**Comment by HerrDerb** [comment]
@cpanato Thank you so much for your non-self-evident super quick work! 😀🥳

**Comment by HerrDerb** [comment]
I guess we can close this as it is released and working with 2.6.1
